inst/pages/integrated_learner.qmd

```diff
@@ -19,161 +17,203 @@ advanced method, called cooperative learning [@Ding2022], which is also known as
+model.
 
-```{r prepare-metadata-feature}
+We can visualize the classification performance by constructing layer-wise ROC 
+curves for both train and test set using plot.learner() function that takes 
+IntegratedLearner object as input.
 
-##########################
-# Create sample metadata #
-##########################
+```{r plot fit1}
+###################
+# Summary outputs #
+###################
 
-sample_metadata <- select(
-    as.data.frame(colData(se_relative)),
-    c("study_name", "disease", "subject_id"))
+plot <- IntegratedLearner:::plot.learner(fit1)
 
-# Define response variable & sample id
-sample_metadata$Y <- ifelse(sample_metadata$disease == "IBD", 1, 0)
-sample_metadata <- sample_metadata %>%
-    select(subject_id, Y) %>% rename(subjectID = subject_id)
+```
 
-###########################
-# Create Species Features #
-###########################
+The stacked model achieves superior accuracy to individual layers and concatenation 
+model in independent validation, indicating that the stacked multi-omics classifier 
+leads to a competitive or superior cross-validated and independent validation 
+classification accuracy than its single-omics counterparts.
 
-feature_species <- as.data.frame(assay(se_relative))
-rownames(feature_species) <- sub('.*s__', '', rownames(feature_species))
+### Without validataion set
 
-###########################
-# Create Pathway Features #
-###########################
+The default base model recommendation for IntegratedLearner is Bayesian additive 
+regression trees or BART (base_learner='SL.BART'). Using BART as base-model yields 
+uncertainty estimates (i.e. credible intervals) of the prediction and model 
+parameters in addition to reporting a small set of interpretable features for 
+follow-up experiments (i.e. feature importance scores).
 
-feature_pwys <- as.data.frame(assay(se_pathway))
-feature_pwys <- rownames_to_column(feature_pwys, "ID")
-feature_pwys <- feature_pwys %>%
-  filter(!grepl("\\|", ID)) %>%
-  filter(!ID %in% c('UNMAPPED', 'UNINTEGRATED')) %>%
-  column_to_rownames('ID') %>%
-  as.data.frame()
+```{r no_valid set}
 
-################################
-# Combine Species and Pathways #
-################################
+##############################
+# Run without validation set #
+##############################
 
-feature_table <- bind_rows(feature_species, feature_pwys)
+fit2 <- IntegratedLearnerFromMAE(
+    mae,
+    experiment = c(1, 2, 3),
+    assay.type = c("counts", "nmr", "signals"),
+    outcome.col = "Diet",
+    folds = 10,
+    na.rm = TRUE # The data has missing values that are removed
+)
 
-########################################################
-# Check row names of feature_table and sample_metadata #
-########################################################
+```
+
+```{r plot fit2}
+
+###################
+# Summary outputs #
+###################
 
-all(colnames(feature_table) == rownames(sample_metadata))
+plot <- IntegratedLearner:::plot.learner(fit2)
```

Comment:
Use functions without specifying a package (consistency)

Reply:
plot.learner is not exported.